gateway/server.go

```diff
@@ -1245,6 +1245,9 @@ func (gw *Gateway) initialiseSystem() error {
 	overrideTykErrors(gw)
 
 	gwConfig := gw.GetConfig()
+	// Initialize the appropriate log formatter
+	log.Formatter = logger.NewFormatter(gwConfig.LogFormat)
```

Comment:
```suggestion
	if gwConfig.LogFormat != "" {
		log.Formatter = logger.NewFormatter(gwConfig.LogFormat)
	}
```

Reply:
This should keep the formatter from resetting whatever value was configured with the `TYK_LOGFORMAT` env (testing etc.).